assets/scripts/dialogs/__tests__/GeotagDialog.test.js

```diff
@@ -2,9 +2,53 @@
 import React from 'react'
 import GeotagDialog from '../GeotagDialog'
 import { shallow } from 'enzyme'
+import { getRemixOnFirstEdit } from '../../streets/remix'
 
 // Mock dependencies that could break tests
-jest.mock('../../streets/remix', () => {})
+jest.mock('../../streets/remix', () => ({
+  getRemixOnFirstEdit: jest.fn()
+}))
+
+function getTestComponent (addressInformation = null, street = null) {
```

Comment:
The arguments will have the value `undefined` by default if nothing gets passed in, so you don't need to set it to a default value of `null` -- it will still evaluate to falsy.